lib/lambda/getUploadUrl.test.ts

```diff
@@ -79,9 +77,14 @@ describe("Handler for generating signed URL", () => {
     });
   });
 
-  it("should throw an error if required environment variables are missing", () => {
+  it("should throw an error if required environment variables are missing", async () => {
     delete process.env.attachmentsBucketName;
 
-    expect(() => handler({} as APIGatewayEvent)).toThrowError;
```

Comment:
This test never ran as it was never called – eslint caught the error and I fixed it

Reply:
`handler` can't have thrown an error because it was being mocked, so we had to spy on the `response` function like we did in the previous tests